workspaces/github-actions/plugins/github-actions/src/components/getProjectNameFromEntity.ts

```diff
@@ -19,5 +19,8 @@ import { Entity } from '@backstage/catalog-model';
 /** @public */
 export const GITHUB_ACTIONS_ANNOTATION = 'github.com/project-slug';
 
+export const hasGithubAnnotation = (entity: Entity) =>
```

Comment:
Would we want to use this instead? 

https://github.com/backstage/community-plugins/blob/6cc64e66ae10474f02e0179584abcd483dafb268/workspaces/github-actions/plugins/github-actions/src/components/Router.tsx#L32

Reply:
great catches! thank you 🙏